app/models/pydantic/creation_options.py

```diff
@@ -228,6 +228,7 @@ class TableAssetCreationOptions(StrictBaseModel):
         "when geographic columns are present. "
         "Disable this option by setting value to `false`",
     )
+    timeout: int = DEFAULT_JOB_DURATION
```

Comment:
Since you add a default values here this should work, but please double check that we don't get an error when retrieving an old creation option where the value was not yet set.

Reply:
Looks like it works, just adds the default value